pkg/framework/plugins/removepodsviolatingtopologyspreadconstraint/topologyspreadconstraint_test.go

```diff
@@ -63,12 +67,9 @@ func TestTopologySpreadConstraint(t *testing.T) {
 				},
 			}),
 			expectedEvictedCount: 0,
-			strategy: api.DeschedulerStrategy{
-				Params: &api.StrategyParameters{
-					NodeFit: false,
-				},
-			},
-			namespaces: []string{"ns1"},
+			namespaces:           []string{"ns1"},
+			args:                 componentconfig.RemovePodsViolatingTopologySpreadConstraintArgs{},
+			nodeFit:              false,
```

Comment:
This line can be omitted as `false` is the default value for `nodeFit`. The same for other occurrences.